pom.xml

```diff
@@ -42,7 +43,7 @@
 
         <!-- environment specific variables-->
         <accumulo.version>1.7.2</accumulo.version>
-        <zookeeper.version>3.4.5</zookeeper.version>
+        <zookeeper.version>3.4.9</zookeeper.version>
```

Comment:
i updated it b/c i was seeing some issues (although it didn't help...). i think the gm version is just what we have cq'd, but in general you'd want the latest compatible.